lib/utils/client-helpers.ts

```diff
@@ -1,52 +1,47 @@
+export type ChainModifiers =
+  | 'allLocales'
```

Comment:
If understood correctly, this is only used for discrimination, right? In this case, we could also think about using Capitalized Case strings, wdyt?

And is there a specific reason why not stick with the existing names ("including the "with")?

Reply:
We could also use capitalized modifiers. I’ll check separately what is best practice for these.

The main reason not to use the existing names was to keep it short. For the user interface the `with` version is nice to read but in a long list of conditional expressions with all combinations it helps with readability IMO.